rust_language/generate.go

```diff
@@ -38,6 +38,8 @@ func (l *rustLang) inferRuleKind(filename string, dirname *string,
 		return "rust_binary"
 	} else if filename == "lib.rs" {
 		return "rust_library"
+	} else if filename == "build.rs" {
+		return "cargo_build_script"
```

Comment:
This path is only used for pure-bazel mode. It's not clear to me that this will do the right thing by default, so I would prefer to make this change separately and test it.